lib/frontier/attribute/validation/uniqueness.rb

```diff
@@ -0,0 +1,60 @@
+class Frontier::Attribute::Validation::Uniqueness < Frontier::Attribute::Validation
+
+  # attribute: name
+  # key: "uniqueness"
+  # args: true OR {scope: :user_id}
+  def initialize(attribute, key, args)
+    super
+
+    # For a scope: argument, ensure the value is :value
+    if args.is_a?(Hash) && args["scope"] && args["scope"].is_a?(String)
+      args["scope"] = args["scope"].include?(":") ? args["scope"] : ":" + args["scope"]
+    end
+  end
+
+  def as_spec
+    if args.present?
+      raw = <<-STRING
+describe "validating uniqueness" do
+  subject { FactoryGirl.create(#{attribute.model_configuration.as_symbol}) }
+  it { should #{full_assertion} }
+end
+STRING
```

Comment:
Peep the [jewels](http://commandercoriander.net/blog/2014/11/09/a-multiline-string-cheatsheet-for-ruby/)